include/swift/SIL/SILFunctionBuilder.h

```diff
@@ -79,7 +79,9 @@ class SILFunctionBuilder {
       IsDynamicallyReplaceable_t isDynamic,
       ProfileCounter entryCount = ProfileCounter(),
       IsThunk_t isThunk = IsNotThunk,
-      SubclassScope subclassScope = SubclassScope::NotApplicable);
+      SubclassScope subclassScope = SubclassScope::NotApplicable,
+      // SWIFT_ENABLE_TENSORFLOW
```

Comment:
Why not just call ->setInlineStategy()?

Reply:
Done in https://github.com/apple/swift/pull/27754/commits/0e7f03f5c79a5a3d17a227a52e33e5c803c880df.